advanced/keyboard-shortcuts.rst

```diff
@@ -4,45 +4,136 @@ Keyboard Shortcuts
+``shift`` + ``ctrl`` + ``h``          Show list of shortcuts
+``shift`` + ``ctrl`` + ``w``          Close current tab
+``shift`` + ``ctrl`` + ``tab``        Show next tab
+``shift`` + ``ctrl`` + ``shift+tab``  Show previous tab
+``shift`` + ``ctrl`` + ``return``     Confirm/submit in dialogs
+``↑`` / ``↓``                         Move selection/cursor up and down
+``←`` / ``→``                         Move selection/cursor left and right
+``enter``                             Select item / confirm
+``shift`` + ``ctrl`` + ``.``          Copy current object number (e.g. Ticket number)
+``shift`` + ``ctrl`` + 2x ``.``       Add the title of the object to the number
+``shift`` + ``ctrl`` + 3x ``.``       Add the object link URL to the number and title
+====================================  ===================================================
+
+
+Translations
+^^^^^^^^^^^^
+Note: you need to have admin permissions to enable it.
+
+====================================  ================================================
+Key / key combination                       Function
+====================================  ================================================
+``shift`` + ``ctrl`` + ``t``          Enable or disable the inline translation
+====================================  ================================================
+
+
+Appearance
+^^^^^^^^^^
+
+====================================  ================================================
+Key / key combination                       Function
+====================================  ================================================
+``d``                                 Switch between dark and light mode
+====================================  ================================================
+
+Tickets
+^^^^^^^
+
+====================================  ===================================================================
+Key / key combination                 Function
+====================================  ===================================================================
+``shift`` + ``ctrl`` + ``m``          Create a new note article
+``shift`` + ``ctrl`` + ``g``          Reply to the last article
+``shift`` + ``ctrl`` + ``j``          Switch the visibility of the article between internal and public
+``shift`` + ``ctrl`` + ``c``          Set state of the ticket to "closed"
+``shift`` + ``ctrl`` + ``←`` / ``→``  Navigate through article
+``::``                                Insert text module (while composing an article)
+``??``                                Insert knowledge base article (while composing an article)
+``@@``                                Mention a user (while composing an article)
+====================================  ===================================================================
+
+Text editing
+^^^^^^^^^^^^
+
+.. tip::
+   You can apply a text format *before* typing or *after* typing.
+
+   Before typing:
+
+   * Press ``cmd`` + ``i`` to enter Italics mode,
```

Comment:
italics. You can highlight it - if needed - with italic formatting, just as 'normal text mode' below?